<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_main_qr_code_over_view_cash_pay</class>
 <widget class="QDialog" name="w_scr_dish_main_qr_code_over_view_cash_pay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="lds_label_doublespinbox" name="lineEdit_cost" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_label_doublespinbox" name="lineEdit_round" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_label_doublespinbox" name="lineEdit_cash" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_label_doublespinbox" name="lineEdit_money" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="lds_label_doublespinbox" name="lineEdit_change" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="public_pushbutton_numkey" name="pushButton_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="public_pushbutton_numkey" name="pushButton_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="public_pushbutton_numkey" name="pushButton_16">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>.</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="public_pushbutton_numkey" name="pushButton_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="public_pushbutton_numkey" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="public_pushbutton_numkey" name="pushButton_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="public_pushbutton_numkey" name="pushButton_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="public_pushbutton_numkey" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="public_pushbutton_numkey" name="pushButton_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="public_pushbutton_numkey" name="pushButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="public_pushbutton_numkey" name="pushButton_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>50</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="public_pushbutton_numkey" name="pushButton_17">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>100</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="public_pushbutton_numkey" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="public_pushbutton_numkey" name="pushButton_15">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="public_pushbutton_numkey" name="pushButton_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="public_pushbutton_numkey" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>10</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton_key</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>public_pushbutton_numkey</class>
   <extends>QPushButton</extends>
   <header>public_pushbutton_numkey.h</header>
  </customwidget>
  <customwidget>
   <class>lds_label_doublespinbox</class>
   <extends>QWidget</extends>
   <header>lds_label_doublespinbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
